### PR TITLE
[5.1] Update VerifyCsrfToken.php

### PR DESCRIPTION
**Pull request by akaroot**
Use cookie secure attribute from config session.secure

### CODE REVIEW

File: src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php [hunk 1]
```diff
@@ -111,7 +111,7 @@ protected function addCookieToResponse($request, $response)
         $response->headers->setCookie(
             new Cookie(
                 'XSRF-TOKEN', $request->session()->token(), time() + 60 * 120,
-                $config['path'], $config['domain'], false, false
+                $config['path'], $config['domain'], $config['secure'], false
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php]: So annoying. :sob:

**Comment by akaroot** [comment]
Should I make the same pull request for 5.2 ?

**Comment by GrahamCampbell** [comment]
No.

**Comment by GrahamCampbell** [comment]
We'll merge 5.1 -> 5.2 -> master on a fairly regular basis.

**Comment by GrahamCampbell** [comment]
Always target PRs to the lowest version, and it'll end up in all subsequent versions when we do branch merges. ;)

**Comment by akaroot** [comment]
Thanks

**Pull request merged**